src/server/api/controllers/users.controller.js

```diff
@@ -1,18 +1,11 @@
 /* eslint-disable camelcase */
 const knex = require('../../config/db');
-const HttpError = require('../lib/utils/http-error');
 
 const getUsers = async () => {
   return knex('users');
 };
 
 const getUsersById = async (id) => {
-  if (!id) {
-    throw new HttpError(
-      'Bad request. User ID must be an integer and larger than 0',
-      400,
-    );
-  }
```

Comment:
We still need guard statements to check if there is an ID. Currently you are relying on the database to fail to tell you if it does not work. If you pass in an empty string or an integer you could have avoided making the request at all by having a guard statement.

Reply:
Write a check that makes sure that ID is a string and it is not an empty string or undefined and use that to set up guard statements.